src/components/MultiAccordion/MultiAccordion.tsx

```diff
@@ -0,0 +1,286 @@
+import * as RadixAccordion from "@radix-ui/react-accordion";
+import styled from "styled-components";
+import { IconSize } from "@/components/Icon/types";
+import { Container, Icon, IconName, Spacer, Text } from "@/components";
+import {
+  createContext,
+  MouseEventHandler,
+  ReactElement,
+  ReactNode,
+  useContext,
+} from "react";
+import { GapOptions, PaddingOptions } from "../Container/Container";
+import { SizeType as SpacerSizeType } from "../Spacer/Spacer";
+
+type Size = "sm" | "md" | "lg";
+type Color = "default" | "link";
+
+type MarkAsCompletedFunctionType = (value: string) => void | Promise<void>;
+
+interface MultiAccordionCommonProps {
+  children: React.ReactNode;
+  size?: Size;
+  fillWidth?: boolean;
+  gap?: GapOptions;
+  padding?: PaddingOptions;
+  showBorder?: boolean;
+  showCheck?: boolean;
+  markAsCompleted?: MarkAsCompletedFunctionType;
+}
+
+export type MultiAccordionProps = MultiAccordionCommonProps &
+  (
+    | Omit<RadixAccordion.AccordionMultipleProps, "children">
+    | Omit<RadixAccordion.AccordionSingleProps, "children">
+  );
+
+interface MultiAccordionContextProps {
+  size: Size;
+  fillWidth: boolean;
+  padding: PaddingOptions;
+  showBorder: boolean;
+  showCheck: boolean;
+  markAsCompleted?: MarkAsCompletedFunctionType;
+}
+
+const MultiAccordionContext = createContext<MultiAccordionContextProps>({
+  size: "md",
+  fillWidth: true,
+  padding: "md",
+  showBorder: true,
+  showCheck: false,
+});
+
+export const MultiAccordion = ({
```

Comment:
Why do we call it MultiAccordion?

Reply:
We call it multi accordion because the accordion supports single and multi type option
Since our normal accordion only supports the single option I named it multiAccordion

Reply:
Can't this all be abstracted into just 1 single accordion? sounds like the single option is basically a specialization of this one.